lxd/storage/drivers/utils.go

```diff
@@ -211,6 +211,20 @@ func tryExists(path string) bool {
 	return false
 }
 
+// waitGone waits up to 10s for a file to not exist anymore.
+func waitGone(path string) bool {
```

Comment:
Can we have this accept a context so the timeout can be customized and cancelled if needed?

Reply:
Yes sure, I'll add the same for `tryExists` a few lines below.

Reply:
Fixed.